pkg/config/kind.go

```diff
@@ -35,24 +35,24 @@ const (
 
 // Name of all supported aspect kinds.
 const (
-	AccessLogsKindName          = "access-logs"
-	ApplicationLogsKindName     = "application-logs"
-	AttributeGenerationKindName = "attribute-generation"
-	DenialsKindName             = "denials"
-	ListsKindName               = "lists"
-	MetricsKindName             = "metrics"
-	QuotasKindName              = "quotas"
+	AccessLogsKindName      = "access-logs"
+	ApplicationLogsKindName = "application-logs"
+	AttributesKindName      = "attributes"
```

Comment:
What about "preprocessors" or something in that vein? I dislike "attributes" because, as you point out, I think it's likely to cause confusion. Plus, it's not strictly required that an adapter running in the preprocess phase generates attributes.